setup.py

```diff
@@ -111,6 +111,8 @@ def get_release_info():
 ]
 
 entry_points = {'console_scripts': console_scripts,
+                'taurus.qt.qtgui.panel.TaurusModelSelector.items':
+                    ['Tango = taurus.qt.qtgui.panel:TangoModelSelectorItem',]
```

Comment:
See previous comment about not exporting `TangoModelSelectorItem` in  `taurus.qt.qtgui.panel` and use instead:

 `'Tango = taurus.qt.qtgui.panel.taurusmodelchooser:TangoModelSelectorItem'`

Reply:
Done